docs/design/solutions/pce-weights-strategies/pce-weights-strategies.md

```diff
@@ -1,23 +1,22 @@
-# Evolve PCE with weights computation strategies
+# PCE weight computation strategies
 
 ## Goals
-For now PCE uses only cost field as graph edges weights
 
 While path computing logic going to be more complicated - we need to take several factors\costs to compute final edge\node weight, and keep this things maintainable.
-Instead of current "computer class per cost strategy", proposed to use one computer class parameterized with "get weight" strategy function.
+One path computer class parameterized with "get weight" strategy function is introduced instead of "computer class per cost strategy" design.
 
-## Proposed changes
-- Introduce PCE data model instead of reusing DAO: for storing graph weighs by features and more convenient logging
-- Decouple AvailableNetwork creation: it will use own building strategies.
-- Introduce weights computing strategies: like weight getters above data model, defines what weigh features will be used in path computation and in that proportions
+## Features
+- Introduced PCE data model instead of reusing DAO: for storing graph weighs by features and more convenient logging
+- Decoupled AvailableNetwork creation: it has own building strategies.
+- Introduced weight computing strategies: edge data model to long functions, defines what weigh features will be used in path computation and in what proportions
```

Comment:
I believe this should be covered in details

Reply:
Added some details. Note: it's already done features. I just reworked this doc from "todo-style" to "done-style".